File: app-server-webflux/src/main/java/com/github/arhor/simple/expense/tracker/service/mapper/SharedMappingConfig.java
package com.github.arhor.simple.expense.tracker.service.mapper;

import org.mapstruct.MapperConfig;
import org.mapstruct.NullValueMappingStrategy;

@MapperConfig(
    componentModel = "spring",
    nullValueMappingStrategy = NullValueMappingStrategy.RETURN_DEFAULT,
    uses = OptionalMapper.class
)
public final class SharedMappingConfig {}

File: app-server/src/main/kotlin/com/github/arhor/simple/expense/tracker/service/mapping/SharedMappingConfig.kt
package com.github.arhor.simple.expense.tracker.service.mapping;

import org.mapstruct.MapperConfig;
import org.mapstruct.NullValueMappingStrategy;

@MapperConfig(
    componentModel = "spring",
    nullValueMappingStrategy = NullValueMappingStrategy.RETURN_DEFAULT,
    uses = [OptionalMapper::class],
)
class SharedMappingConfig
